.github/workflows/ci-test.yml

```
@@ -108,6 +121,23 @@ jobs:
             sleep 3
           done
 
+      - name: Elasticsearch 체크
+        run: |
+          until curl -sSf http://localhost:9200/_cluster/health; do
+            echo "Waiting for Elasticsearch..."
+            sleep 5
+          done
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)